wolfssl/wolfcrypt/types.h

```diff
@@ -51,26 +51,30 @@
 
 
 	/* try to set SIZEOF_LONG or LONG_LONG if user didn't */
-	#if !defined(_MSC_VER) && !defined(__BCPLUSPLUS__)
+	#if !defined(_MSC_VER) && !defined(__BCPLUSPLUS__) && !defined(__EMSCRIPTEN__)
 	    #if !defined(SIZEOF_LONG_LONG) && !defined(SIZEOF_LONG)
-	        #if (defined(__alpha__) || defined(__ia64__) || defined(_ARCH_PPC64) \
-	                || defined(__mips64)  || defined(__x86_64__) || \
-                    ((defined(sun) || defined(__sun)) && \
+	        #if (defined(__alpha__) || defined(__ia64__) || \
+                   defined(_ARCH_PPC64) || defined(__mips64) || \
+                   defined(__x86_64__) || \
+                   ((defined(sun) || defined(__sun)) && \
                      (defined(LP64) || defined(_LP64))))
 	            /* long should be 64bit */
 	            #define SIZEOF_LONG 8
-	        #elif defined(__i386__) || defined(__CORTEX_M3__)
-	            /* long long should be 64bit */
-	            #define SIZEOF_LONG_LONG 8
+            #elif (defined(__i386__) || defined(__CORTEX_M3__)
```

Comment:
There is a wild paren ( here... I'll push a fix. Not sure why the compiler didn't report an issue...